### PR TITLE
(fluid清理）remove stack in nn.py under fluid

### PR DESCRIPTION
**Pull request by Courtesy-Xs**
### PR types

Others
### PR changes

Others
### Describe

删除paddle.fluid.layers.stack接口，后续组网可使用paddle.stack接口替换

### CODE REVIEW

**Review by Xreki — APPROVED** [review]
LGTM

**Review by momozi1996 — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

File: python/paddle/fluid/layers/tensor.py [hunk 1]
```diff
@@ -571,8 +571,9 @@ def tensor_array_to_tensor(input, axis=1, name=None, use_stack=False):
         assert isinstance(
             input, list
         ), "The 'input' in tensor_array_to_tensor must be list"
-        from .nn import stack, concat
+        from .nn import concat
```
Comment (JiabinYang) [inline python/paddle/fluid/layers/tensor.py:574]: plz，add to do to mark import here may cause performance issue

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**